shared/resources/styles/components/notifications-bar.scss

```diff
@@ -32,6 +32,10 @@
       flex: 1;
     }
 
+    .mobile-text {
+      display: none;
+    }
+
```

Comment:
looks good. I was wondering if we can put this in the `global/notifications-bar.scss` file?

Reply:
That's there in the shared one so it won't show up on the exercises screens. I thought about putting it all into shared, but I don't think it gets the responsive vars